pandas/core/dtypes/dtypes.py

```diff
@@ -1099,7 +1100,7 @@ class IntervalDtype(PandasExtensionDtype):
 
     _cache_dtypes: dict[str_type, PandasExtensionDtype] = {}
 
-    def __new__(cls, subtype=None, closed: str_type | None = None):
+    def __new__(cls, subtype=None, closed=None):
```

Comment:
Could you add it here too?

Reply:
Adding ``# type: ignore[assignment] `` in 1141 would work here. This isn't ideal, but it's better if the function signature is typed more strictly

Reply:
I added what you asked but I have a check that fails, I don't know why..
Do you have an idea of what's the reason?